pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -447,14 +452,25 @@ func (gce *GCECloud) EnsureLoadBalancerDeleted(clusterName string, service *v1.S
 	glog.V(2).Infof("EnsureLoadBalancerDeleted(%v, %v, %v, %v, %v)", clusterName, service.Namespace, service.Name, loadBalancerName,
 		gce.region)
 
-	var hc *compute.HttpHealthCheck
+	hcNames := []string{}
```

Comment:
Nit: In every employer of mine that used Go, this way was more preferable: 
`var hcNames []string`

`append` handles the initialization.